extension/src/dashboard/pages/Bookmarks/SubscriptionsTab/SubscriptionsByInformationList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import useAxiosPaginated from 'src/hooks/use-axios-paginated';
+import ReactionsByInformationList from 'src/dashboard/components/ReactionsByInformationList';
+
+import { parseSubscription } from 'src/types/Subscription';
+
+const SubscriptionsByInformationList: React.FC = () => {
+  const { id } = useParams();
```

Comment:
```suggestion
  const { id } = useParams<{ id: string }>();
```
(not sure that's possible)